.github/workflows/test.yml

```
@@ -18,7 +18,7 @@ jobs:
           go-version: 'stable'
 
       - name: Run Go unit tests
-        run: go test -C ./tools .
+        run: go test -C ./tools -v .
```

Out of curiosity, any reason to not have:

```suggestion
        run: cd ./tools && go test -v ./...
```

---

Instead of chaining a `cd`, what about using the [working-directory](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory) job config element?

---

I don't have a strong opinion on how this is done. Up to maintainers to decide this. My main opinion is this is currently breaking the pipeline, so either we rollback #1901, or we accept this as is and revisit it if the need arises?

I do have a preference for using the tools that Go has built in to the toolchain.